docs/src/rules/prefer-arrow-callback.md

```diff
@@ -110,6 +110,36 @@ someArray.map(function(item) { return this.doSomething(item); }, someObject);
 
 :::
 
+This rule additionally supports TypeScript type syntax. However, it does not autofix callback `functions` that use `this` as a parameter.
```

Comment:
We don't have to mention this. It's expected behavior.

```suggestion
This rule additionally supports TypeScript type syntax.
```

Reply:
Oh funny I'd suggested something opposite-ish: https://github.com/eslint/eslint/pull/19678#discussion_r2071620253

Maybe just including `this` in the code sample?